tests/e2e/operator.sh

```diff
@@ -132,12 +135,12 @@ install_ccruntime() {
 	local pod=""
 	local cmd=""
 	for pod in cc-operator-daemon-install cc-operator-pre-install-daemon; do
-		cmd="kubectl get pods -n "$op_ns" --no-headers |"
+		cmd="kubectl get pods -n '$op_ns' --no-headers |"
 		cmd+="egrep -q ${pod}.*'\<Running\>'"
 		if ! wait_for_process 600 30 "$cmd"; then
 			echo "::error:: $pod pod is not running"
-
-			local pod_id="$(get_pods_regex $pod $op_ns)"
+			local pod_id
+			pod_id="$(get_pods_regex '$pod' '$op_ns')"
```

Comment:
^^ unlike variable substitution in shell execution the expansion starts over inside `$( ... )` therefore here you need to use `"` to expand the variables.